scripts/apps/ingest/services/SendService.ts

```diff
@@ -212,10 +212,13 @@ export function SendService(
                     $rootScope.$broadcast('item:update', {item: _item});
                     return _item;
                 });
-        } else if (!item.lock_user) {
+        } else {
             return api.save('move', {}, {task: data, allPackageItems: config.sendAllPackageItems}, item)
                 .then((_item) => {
                     $rootScope.$broadcast('item:update', {item: _item});
+                    if (item.lock_user) {
+                        getAuthoringWorkspace().close();
+                    }
                     if (config.open) {
```

Comment:
what does config.open mean? When is it true? Would it be compatible with you closing the item in the line before?

Reply:
`config.open` means , if user select option send and open, so in this case first we move that item then open it.

![Superdesk - Monitoring - desk testing](https://user-images.githubusercontent.com/73937490/222708265-94b520e6-0662-4df5-b752-177025f66b07.png)